packages/fxa-js-client/client/FxAccountClient.js

```diff
@@ -836,6 +836,27 @@ define([
       });
   };
 
+  /**
+   * Gets the account profile
+   *
+   * @method accountProfile
+   * @param {String} sessionToken User session token
+   * @return {Promise} A promise that will be fulfilled with JSON `xhr.responseText` of the request
+   */
+  FxAccountClient.prototype.accountProfile = function(sessionToken) {
```

Comment:
This LGTM, longer term think we should have this method and other OAuth related methods accept either a sessionToken or OAuth creds?

Reply:
yeah we can expand on that